### PR TITLE
API Update deprecations

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10531

### CODE REVIEW

**Review by GuySartorelli — APPROVED** [review]

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Versions are incorrect - use `1` major instead.
This will need to be applied to any other deprecations in this module, as well.

File: _legacy/GraphQL/ReadOneLegacyResolver.php [hunk 1]
```diff
@@ -17,7 +18,7 @@
 /**
  * Shim to make readOne work like GraphQL 4
  *
- * @deprecated 4.8..5.0 Use silverstripe/graphql:^4 functionality.
+ * @deprecated 1.8.0 Use _graphql directory functionality instead
```
Comment (ChloeSartorelli) [inline _legacy/GraphQL/ReadOneLegacyResolver.php]: Sorry, I should have caught this the first time around. This should say something like "use the latest version of silverstripe/graphql" - the directory isn't the point.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: _legacy/GraphQL/ReadOneLegacyResolver.php (hunk 1 above)
Comment (emteknetnz) [reply _legacy/GraphQL/ReadOneLegacyResolver.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
MOG

**Pull request merged**